ui/View.elm

```diff
@@ -145,7 +164,8 @@ viewDownloadDetails download =
                 [ tr []
                     [ td [] [ a [ href download.url ] [ text filename ] ]
                     , td [] [ text <| Maybe.withDefault "" download.mimetype ]
-                      -- TODO display the size in an humanly readable format
+
+                    -- TODO display the size in an humanly readable format
```

Comment:
Note: @magopian I had the same formatting, you should update elm-format to the latest version.